src/cultionet/losses/topological.py

```diff
@@ -0,0 +1,291 @@
+import typing as T

```

Comment:
This file is associated with the topological loss, but is not currently used anywhere.